### ISSUE TITLE
Configure fails to find installed SDK on 10.6 with Xcode 3.2: `No supported macOS SDK installed`

### ISSUE THREAD

**Issue by barracuda156** [osxfuse/osxfuse#1037]
Configure fails to recognize installed SDK with Xcode 3.2:
```
--->  Configuring osxfuse
--->  Building osxfuse
Executing:  cd "/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_fuse_osxfuse/osxfuse/work/osxfuse-osxfuse-48bc246" && ./build.sh -t packagemanager -a build -v 5 --build-directory="/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_fuse_osxfuse/osxfuse/work" -- -a ppc --framework-prefix="/opt/local" --fsbundle-prefix="/opt/local" --library-prefix="/opt/local" 
DEFAULT_BUILD_CONFIGURATION=Release
DEFAULT_BUILD_DIRECTORY=/tmp/osxfuse
DEFAULT_LOG_VERBOSE=2
DEFAULT_PREFIX=''
DEFAULT_SDK=10.6
DEFAULT_SDK_10_10_ARCHITECURES=([0]=i386 [1]=x86_64)
DEFAULT_SDK_10_10_COMPILER=com.apple.compilers.llvm.clang.1_0
DEFAULT_SDK_10_11_ARCHITECURES=([0]=i386 [1]=x86_64)
DEFAULT_SDK_10_11_COMPILER=com.apple.compilers.llvm.clang.1_0
DEFAULT_SDK_10_12_ARCHITECURES=([0]=i386 [1]=x86_64)
DEFAULT_SDK_10_12_COMPILER=com.apple.compilers.llvm.clang.1_0
DEFAULT_SDK_10_13_ARCHITECURES=([0]=i386 [1]=x86_64)
DEFAULT_SDK_10_13_COMPILER=com.apple.compilers.llvm.clang.1_0
DEFAULT_SDK_10_14_ARCHITECURES=([0]=x86_64)
DEFAULT_SDK_10_14_COMPILER=com.apple.compilers.llvm.clang.1_0
DEFAULT_SDK_10_5_ARCHITECURES=([0]=ppc [1]=ppc64 [2]=i386 [3]=x86_64)
DEFAULT_SDK_10_5_COMPILER=4.2
DEFAULT_SDK_10_6_ARCHITECURES=([0]=ppc [1]=i386 [2]=x86_64)
DEFAULT_SDK_10_6_COMPILER=4.2
DEFAULT_SDK_10_7_ARCHITECURES=([0]=i386 [1]=x86_64)
DEFAULT_SDK_10_7_COMPILER=com.apple.compilers.llvmgcc42
DEFAULT_SDK_10_8_ARCHITECURES=([0]=i386 [1]=x86_64)
DEFAULT_SDK_10_8_COMPILER=com.apple.compilers.llvm.clang.1_0
DEFAULT_SDK_10_9_ARCHITECURES=([0]=i386 [1]=x86_64)
DEFAULT_SDK_10_9_COMPILER=com.apple.compilers.llvm.clang.1_0
DEFAULT_SDK_SUPPORTED=([0]=10.5 [1]=10.6 [2]=10.7 [3]=10.8 [4]=10.9 [5]=10.10 [6]=10.11 [7]=10.12 [8]=10.13 [9]=10.14)
DEFAULT_SOURCE_DIRECTORY=/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_fuse_osxfuse/osxfuse/work/osxfuse-osxfuse-48bc246
BUILD_SOURCE_DIRECTORY=/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_fuse_osxfuse/osxfuse/work/osxfuse-osxfuse-48bc246
BUILD_BUILD_DIRECTORY=/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_fuse_osxfuse/osxfuse/work
Search for Xcode
Xcode 3.2 found at path '/Developer'
XCODE_3_2_PATH=/Developer
XCODE_INSTALLED=([0]=3.2)
XCODE_SDK_INSTALLED=()
Done searching for Xcode
No supported macOS SDK installed
```

Of course, SDKs are present:
```
36-122% xcodebuild -showsdks
Mac OS X SDKs:
	Mac OS X 10.4                 	-sdk macosx10.4
	Mac OS X 10.5                 	-sdk macosx10.5
	Mac OS X 10.6                 	-sdk macosx10.6

36-122% xcodebuild -version
Xcode 3.2
Component versions: DevToolsCore-1534.0; DevToolsSupport-1528.0
BuildVersion: 10A190
```

Something does not work correctly with configure script. On 10.6.8 in Rosetta with Xcode 3.2.6 it worked as expected, so the failure is specific to 10.6 + Xcode 3.2.

**Comment by bfleischer** [comment]
Please use the latest binary release still supporting your hardware. Xcode 3.2.6 and Snow Leopard are severely outdated.

**Comment by barracuda156** [comment]
@bfleischer 10.6 is the last OS to run on PowerPC. Maybe you could suggest what fails here with the build?

**Comment by bfleischer** [comment]
I've never built osxfuse on a PPC Mac. All releases supporting PPC Macs have been built on an Intel Mac.

**Comment by barracuda156** [comment]
@bfleischer I am pretty sure this is not an arch-specific problem as such. (I can confirm that in a few days when get back to my i386 hardware.)
I think the issue may be that expected version of OS does not match what gets returned. On 10.6 ppc, for example, Darwin version is 10.0.0d2 (this creates issues occasionally).

A local fix could be simply to disable the check which fails: I know I have both SDK and Xcode which are working fine. When I tried, I did not find an easy way to remove the check, since something else started failing, which used the results of that check.

I would appreciate an advice here. It is not needed to introduce any changes specifically to support pre-release 10.6 versions here in upstream. (We may get 10.6.8 working on PowerPC soon, but currently there are some issues, so I have to stay on 10a190 build, which is fully functional.)
